docs/samples/transformer/image_transformer/README.md

```diff
@@ -0,0 +1,105 @@
+# Predict on a KFService using PyTorch Server and Transformer
+
+Most of model servers expect tensors as input data, so a pre-process step is needed before making the prediction call if user is sending in raw input format. Transformer is
+a service we orchestrated from KFService spec for user implemented pre/post process code. In the [pytorch](../../pytorch/README.md) example we send the prediction endpoint with
+tensor input in this example we add additional pre-process step to allow user sending in raw image data.
+
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/docs/serving/outbound-network-access/)
+
+##  Build transformer image
+
+### Extend Transformer and implement pre/postprocess functions
+```python
+from typing import List, Dict
+from kfserving.transformer import Transformer
+from PIL import Image
+import torchvision.transforms as transforms
+import logging
+import io
+import numpy as np
+import base64
+
+transform = transforms.Compose(
+        [transforms.ToTensor(),
+         transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])
+
+
+def image_transform(instance):
+    byte_array = base64.b64decode(instance['image_bytes']['b64'])
+    image = Image.open(io.BytesIO(byte_array))
+    a = np.asarray(image)
+    im = Image.fromarray(a)
+    res = transform(im)
+    logging.info(res)
+    return res.tolist()
+
+
+class ImageTransformer(Transformer):
+
+    def preprocess(self, inputs: Dict) -> Dict:
+        return {'instances': [image_transform(instance) for instance in inputs['instances']]}
+
+    def postprocess(self, inputs: List) -> List:
+        return inputs
+```
+
+### Build transformer docker image
```

Comment:
```suggestion
### Build Transformer docker image
```

Reply:
We should also hyperlink to the Transformer file here in README, and to the Dockerfile as well